ui/components/forms/flavors/select/index.scss

```diff
@@ -22,3 +22,35 @@
     }
   }
 }
+
+#{$css-prefix}form-element__select-container {
+    position: relative;
+    .#{$css-prefix}select {
+        -moz-appearance: none;
+        -webkit-appearance: none;
+        appearance: none;
+        padding-left: $spacing-x-small;
```

Comment:
`padding-right: $spacing-large`, so text doesn't overlap the custom arrows

you can also combine the paddings

```
padding: {
  left: $spacing-x-small;
  right: $spacing-large;
}
```